server/users/controller.js

```diff
@@ -1,7 +1,53 @@
-export const getUsers = async () => {
-  // noop
+import { db } from '../firestore';
+// import { User, userConverter } from '../models/userModel';
+// import { User } from '../models/userModel';
+
+// get users from firestore
+export const getUsers = async (req, res) => {
+  let uname = req.query.username;
```

Comment:
Preferably use `const` when the value does not change

Also, `const { username } = req.query` for destructuring - no need to rename the variables

Reply:
Yes makes sense. Changed it.